website/docs/r/runtimeconfig_variable.html.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: "google"
+page_title: "Google: google_runtimeconfig_variable"
+sidebar_current: "docs-google-runtimeconfig-variable"
+description: |-
+  Manages a RuntimeConfig variable in Google Cloud.
+---
+
+# google\_runtimeconfig\_variable
+
+Manages a RuntimeConfig variable in Google Cloud. For more information, see the
+[official documentation](https://cloud.google.com/deployment-manager/runtime-configurator/),
+or the
+[JSON API](hhttps://cloud.google.com/deployment-manager/runtime-configurator/reference/rest/).
+
+## Example Usage
+
+Example creating a RuntimeConfig variable.
+
+```hcl
+resource "google_runtimeconfig_config" "my-runtime-config" {
+ 	name = "my-service-runtime-config"
+ 	description = "Runtime configuration values for my service"
+}
+
+resource "google_runtimeconfig_variable" "environment" {
+	parent = "${google_runtimeconfig_config.my-runtime-config.name}"
+	name = "environment"
```

Comment:
I think it would be a more realistic example if you were storing a value in a hierarchy instead.
```hcl
...
name = "prod-variables/hostname"
text = "example.com"
```